smbutil.c

```diff
@@ -770,6 +770,7 @@ smb_fdata1(netdissect_options *ndo,
 	    time_t t;
 	    struct tm *lt;
 	    const char *tstring;
+	    char buffer[26];
```

Comment:
"26" must be documented.

Reply:
Done!

Reply:
Using @infrastation hint, I think it's better to use:
`char buffer[sizeof("Www Mmm dd hh:mm:ss yyyy\n")];`